release-notes/3.0/3.0.0-preview2.md

```diff
@@ -0,0 +1,111 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 3.0.0-preview2.21105.01 released 15 April 2021
+
+This update brings the below changes over the previous release:
+
+### Breaking Changes over preview release V3.0.0-preview1
+- `User Id` connection property now accepts `Client Id` instead of `Object Id` for **User-Assigned Managed Identity** [#1010](https://github.com/dotnet/SqlClient/pull/1010)
```

Comment:
```suggestion
- `User Id` connection property now accepts `Client Id` instead of `Object Id` for **User-Assigned Managed Identity** [#1010](https://github.com/dotnet/SqlClient/pull/1010) [Read more](#Azure-Identity-dependency-introduction)
```